site/en/guides/border-router/docker/test-connectivity.md

```diff
@@ -121,28 +121,31 @@ The Thread Commissioning App is not supported.
 ## Step 4: Ping a public address
 
 You should be able to a ping a public IPv4 address from the standalone Thread
-node at this point. Since Thread only uses IPv6, to ping a public IPv4 address
-you must translate it to IPv6 and combine it with the well-known prefix of
-`64:ff9b::/64` used by Network Address Translation (NAT) in OTBR.
-
-1.  To get a translated IPv4 address, use a website like
-    [findipv6.com](https://findipv6.com/ipv4-toipv6/).
-
-1.  Translate the IPv4 address you wish to test. For example, `172.217.164.110`
-    translated to IPv6 is `::ffff:acd9:a46e`.
-
-1.  Using only the last 4 bytes of the resulting IPv6 address, combine it with
-    the well-known prefix of `64:ff9b::/64` to get a new IPv6 address:
-    ```
-    64:ff9b::acd9:a46e
+node at this point. Since Thread only uses IPv6, the public IPv4 address
+will be automatically translated to IPv6 by combining with the NAT64 prefix in
+Thread network.
+
+1.  To view the NAT64 prefix in Thread network
```

Comment:
colon at the end of this, add in "the"

To view the NAT64 prefix in the Thread network: